dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesRun.java

```diff
@@ -257,6 +257,9 @@ public class KubernetesRun extends KubernetesBaseCommand {
                         description = "Maven/Gradle build properties, ex. --build-property=prop1=foo")
     List<String> buildProperties = new ArrayList<>();
 
+    @CommandLine.Option(names = { "--disable-auto" }, description = "Disable automatic cluster type detection.")
+    boolean disableAuto = false;
```

Comment:
i think this shouldn't be backported, it is changing the behavior when a cluster-type is specified

Reply:
makes sense to not add new features, since this is targeted for a patch release.

> Which bug is it fixing?

Probably these two are the main ones, besides the cluster detection which is a new feature.
- camel-jbang-plugin-kubernetes - jkube to generate Deployment instead
- camel-jbang: fix export not respecting --quarkus-group-id parameter